tests/behaviour/steps/concept/thing/relation.rs

```diff
@@ -42,7 +42,7 @@ async fn relation_add_player_for_role(
         {
             let role_type = relates.role();
             let res = relation.add_player(
-                Arc::get_mut(&mut tx.snapshot).unwrap(),
+                tx.snapshot.deref_mut(),
```

Comment:
I think reborrow is cleaner than std::ops.
```suggestion
                &mut *tx.snapshot,
```

Reply:
I don't agree, but my Rust is too below yours

Reply:
I'd leave it this way though, it reads simpler, especially when we talk about tests

Reply:
Nvm, the method got replaced